routes/artifacts.js

```diff
@@ -18,7 +18,7 @@ api.declare({
       'assume:worker-id:<workerGroup>/<workerId>'
     ], [
       'queue:create-artifact:<name>',
-      'queue:claim-task:<taskId>/<runId>'
+      'queue:claim-run-id:<taskId>/<runId>'
```

Comment:
Sorry, docs probably needs an update, but `queue:claim-run-id:<taskId>/<runId>` is granted to the temp credentials returned from `claimTask`.

Hence, whomever has `queue:claim-run-id:<taskId>/<runId>` for a specific taskId and runId, has the claim on that task/run... Hence, it's a worker and workers should always be allowed to upload artifacts, right?